src/API/SSL/SSLClient.php

```diff
@@ -12,6 +12,11 @@ public function getCert($accessToken, $id, $includeChain = false)
         return $this->guzzle($this->getBearerTokenMiddleware($accessToken))->get("ssl/certs/{$id}", ['query' => ['includeChain' => $includeChain]]);
     }
 
+    public function getLetsencryptStatus($accessToken,  $id)
+    {
+        return $this->guzzle($this->getBearerTokenMiddleware($accessToken))->get("/api/ssl/letsencrypt/status/{$id}");
```

Comment:
Fix path for consistency
```suggestion
        return $this->guzzle($this->getBearerTokenMiddleware($accessToken))->get("ssl/letsencrypt/status/{$id}");
```